ctk_cli/module.py

```diff
@@ -259,7 +258,8 @@ def isNumericVector(self):
 
     def isExternalType(self):
         """Return True iff parameter values of this type are transferred via (temporary) files"""
-        return self.typ in self.EXTERNAL_TYPES
+        return (self.typ in self.EXTERNAL_TYPES or
+                self.typ in ['file', 'directory'])
 
```

Comment:
This changes the semantics of this function.  I am not saying that this is bad, but I think that at least the docstring should be adapted slightly.